ompi/mca/osc/rdma/osc_rdma_dynamic.c

```diff
@@ -220,10 +220,16 @@ int ompi_osc_rdma_attach (struct ompi_win_t *win, void *base, size_t len)
                                        (intptr_t) base, module->region_size, &region_index);
 
         if (region_index < region_count) {
-            memmove ((void *) ((intptr_t) region + module->region_size), region,
-                     (region_count - region_index) * module->region_size);
-            memmove (module->dynamic_handles + region_index + 1, module->dynamic_handles + region_index,
+            ret = osc_rdma_accelerator_mem_move ((void *) ((intptr_t) region + module->region_size),
```

Comment:
The `memmove` only happens on host memory (afaics) because that memory is wholly managed by OMPI. The same seems true for the other changes in this file.

Reply:
I can change the memmove and memcpy in the osc_rdma_dynamic file back to the original memmove, I wasn't sure what exactly was and wasn't safe.

Reply:
Changed memmoves and memcpy's in the dynamic file back